Barotrauma/BarotraumaShared/SharedSource/Items/Components/Signal/GreaterComponent.cs

```diff
@@ -25,6 +26,8 @@ public override void Update(float deltaTime, Camera cam)
             if (sendOutput)
             {
                 string signalOut = val1 > val2 ? output : falseOutput;
+                val1 = float.NegativeInfinity;
+                val2 = float.PositiveInfinity;
```

Comment:
As far as I can see, this breaks backwards compatibility. It means the component will not keep sending the same output, but reverts back to false immediately after it's sent one output.